planner/core/cbo_test.go

```diff
@@ -466,46 +465,6 @@ func (s *testAnalyzeSuite) TestOutdatedAnalyze(c *C) {
 	}
 }
 
-func (s *testAnalyzeSuite) TestPreparedNullParam(c *C) {
-	defer testleak.AfterTest(c)()
-	store, dom, err := newStoreWithBootstrap()
-	c.Assert(err, IsNil)
-	defer func() {
-		dom.Close()
-		store.Close()
-	}()
-
-	defer config.RestoreFunc()()
-	flags := []bool{false, true}
-	for _, flag := range flags {
-		config.UpdateGlobal(func(conf *config.Config) {
-			conf.PreparedPlanCache.Enabled = flag
-			conf.PreparedPlanCache.Capacity = 100
-		})
-		testKit := testkit.NewTestKit(c, store)
-		testKit.MustExec("use test")
-		testKit.MustExec("drop table if exists t")
-		testKit.MustExec("create table t (id int not null, KEY id (id))")
-		testKit.MustExec("insert into t values (1), (2), (3)")
-
-		sql := "select * from t where id = ?"
```

Comment:
It will cause panic because it doesn't set the parameters. So the `constant.Eval()` will cause panic. Rewrite it to `TestPreparedNullParam` in `executor/prepared_test.go`.